lib/driver.js

```diff
@@ -182,15 +185,38 @@ class YouiEngineDriver extends BaseDriver {
     let iosArgs = {
       javascriptEnabled: true,
     };
-    let iosdriver = new IOSDriver(iosArgs);
+    let driverName = "IOSDriver";
+    if (opts.platformVersion) {
+      let majorVer = opts.platformVersion.toString().split(".")[0];
```

Comment:
It's mandatory to trigger appium-xcuitest-driver.
If it's not specified it will run appium-ios-driver.
I have created a how-to-doc and will update other areas as well